Dockerfile

```diff
@@ -15,4 +15,4 @@ RUN make PREFIX=/go clean binaries
 VOLUME ["/var/lib/registry"]
 EXPOSE 5000
 ENTRYPOINT ["registry"]
-CMD ["cmd/registry/config.yml"]
```

Comment:
This was a documented mount point. Can't we just move this into place with image build?